src/reflow/mod.rs

```diff
@@ -206,15 +274,65 @@ blanket text without any meaning",
 there are two consecutive newlines \
 in one connected documentation span."] =>
 
-r#" This module contains documentation thats
- is too long for one line and moreover,
- it spans over mulitple lines such that
- we can test our rewrapping algorithm.
- Smart, isn't it? Lorem ipsum and some
- more blanket text without any meaning
+r#"This module contains documentation thats
+is too long for one line and moreover, it
+spans over mulitple lines such that we
+can test our rewrapping algorithm. Smart,
+isn't it? Lorem ipsum and some more
+blanket text without any meaning
+
+But lets also see what happens if there
+are two consecutive newlines in one
+connected documentation span."#);
+    }
+
+    #[test]
+    fn reflow_shorter_than_limit() {
+        reflow!(80 break ["This module contains documentation that is ok for one line"] =>
+                r#" This module contains documentation that is ok for one line"#);
+    }
+
+    #[test]
+    fn reflow_multiple_lines() {
+        reflow!(43 break ["This module contains documentation that is broken",
+                          "into multiple short lines resulting in multiple spans."] =>
+                r#"This module contains documentation that
+is broken into multiple short lines"#);
+    }
+    #[test]
+    fn reflow_indentations() {
+        const CONTENT: &'static str = r#"
+    /// A comment with indentation that spans over
+    /// two lines and should be rewrapped.
+    struct Fluffy {};"#;
+
+        const EXPECTED: &'static str = r#"A comment with indentation
+that spans over two lines
+and should be rewrapped."#;
 
- But lets also see what happens if there
- are two consecutive newlines in one
- connected documentation span."#);
+        let docs = Documentation::from((ContentOrigin::TestEntity, CONTENT));
+        assert_eq!(docs.entry_count(), 1);
+        let chunks = docs
+            .get(&ContentOrigin::TestEntity)
+            .expect("Must contain dummy path");
+        assert_eq!(dbg!(chunks).len(), 1);
+        let chunk = &chunks[0];
+
+        let cfg = ReflowConfig {
+            max_line_length: 35,
+            ..Default::default()
+        };
+        let suggestion_set = reflow(ContentOrigin::TestEntity, chunk, cfg);
+        let suggestions = suggestion_set
+            .iter()
+            .next()
+            .expect("Must contain exactly one item");
```

Comment:
At parity the phrasing is different, but I can't seem to find the coding manual right now (used to be public at least).

Reply:
Why do you use  `qed`? The `expect()` should demonstrate that this never happens. So if the `expect()` is fired, we've shown that this assumption is wrong? More like `quod erat non demonstrandum` :stuck_out_tongue: But anyway, I'll use that phrase :)

Reply:
I'm not sure how to use this style. For instance, on the result of `reflow()` I should do something like `.expect("Reflowing works. qed");`?

Reply:
I think I get it after looking at your examples...